src/ui_parts/viewport.gd

```diff
@@ -71,32 +41,43 @@ func center_frame() -> void:
 func _unhandled_input(event: InputEvent) -> void:
 	if not event is InputEventMouseMotion or event.button_mask != 0:
 		view.queue_redraw()
+	
 	if event is InputEventMouseMotion and\
 	event.button_mask in [MOUSE_BUTTON_MASK_LEFT, MOUSE_BUTTON_MASK_MIDDLE]:
 		set_view(view.position - event.relative)
 	
 	if event is InputEventPanGesture:
 		if event.ctrl_pressed:
-			zoom_level *= 1 + event.delta.y / 2
+			%ZoomMenu.zoom_level *= 1 + event.delta.y / 2
```

Comment:
Create an onready statically typed variable for ZoomMenu.

Reply:
The preference is `@onready` over `@export`?

Reply:
`%ZoomMenu` is a node path reference to a scene-unique node. If I rename ZoomMenu to ZoomWidget, for example, I'll have to change all these references.